### PR TITLE
Write inline documentation for small files

### LINKED ISSUE #326: Inline documentation also for the small files
There are a lot small files without inline documentation. To complete the documentation over all code we should write it.

Files:
* `layer.provider.ts`
* `layer-control.provider.ts`
* `layer-group.provider.ts`
* `map.provider.ts`
* `marker.provider.ts`
* `mouse-event-helper.ts`
* `spec.ts`

### PR DESCRIPTION
**Pull request by atd-schubert**
Closes #326

### CODE REVIEW

File: ts/consts.ts [hunk 1]
```diff
@@ -2,12 +2,39 @@
...
+ */
 export const EXAMPLE_CONTENT: string = 'Vel ipsum odit quia velit omnis illo voluptatem ut. Aperiam porro voluptates maiores.';
+/**
+ * URL-schema of the official osm tile-server
+ */
 export const OSM_TILE_LAYER_URL: string = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+/**
+ * URL for an example image for image-overlays
+ */
 export const IMAGE_OVERLAY_URL: string = 'http://live.osgeo.org/_images/osgeolive_menu6.png';
```
Comment (OpenSteveMap) [inline ts/consts.ts]: URL not working ....

Comment (atd-schubert) [reply ts/consts.ts]: That is not really a problem. We just use it in the unit tests to see that the urls get handled correctly. But we can provide another / better one.

Do you have one?

Comment (OpenSteveMap) [reply ts/consts.ts]: @atd-schubert 
I think that was the pic: [desktop10_osmF32.png](http://download.osgeo.org/livedvd/10.0/desktop10_osmF32.png) (also from osgeo)
or here: [osgeolive_menu.png](https://3.bp.blogspot.com/-GY_wstlz9VE/V6jzgzyatGI/AAAAAAAAA2c/LmIvkvtFNfwXM2Yu6nFWY6Hh_wRNHAhvgCLcB/s1600/osgeolive_menu.png) (not a good source)

**Review by OpenSteveMap — APPROVED** [review]
ok

**Pull request merged**